src/component/RecipeList.js

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+import RecipeListItem from "./RecipeListItem";
+import Pagination from "./Pagination";
+
+function RecipeList({ recipes, handleChooseRecipe }) {
+  const [currentPage, setCurrentPage] = useState(1);
+  //   const [recipeList, setRecipeList] = useState([
+  //     ...recipes.slice(startIndex, endIndex),
+  //   ]);
+  const [recordsPerPage] = useState(10);
+
+  const endIndex = currentPage * recordsPerPage;
+  const startIndex = endIndex - recordsPerPage;
+  const nPages = Math.ceil(recipes.length / recordsPerPage);
```

Comment:
`nPages` is not very descriptive name, maybe `totalPages`?

Reply:
should this be wrapped in useMemo as well?